bug 1490727: Recurring payments user registration styling

joshJarr opened the pull request:
Updates the styles for the user registration page. Enforce text max length and add custom text when the user is redirected from payment flow.

Fixed a small bug it seems, `readable-line-length` was on the form but the class did nothing as `readable-line-length.scss` was scoped to the wiki. I've moved this out as it seems like more like a global utility. - let me know if this is an issue.

**In this PR**
- https://trello.com/c/vrtPOM4e/5-as-a-user-i-can-read-the-payment-terms-so-that-i-can-make-an-informed-decision

**Notes for QA**

- [ ] Ensure user login page does not change site-wide
- [ ] Ensure login page redirects correctly
- [ ] When following recurring payments flow and creating an account there should be text explaining the flow (see screenshot)

**Screenshots**

<img width="670" alt="screenshot 2018-11-05 at 17 32 30" src="https://user-images.githubusercontent.com/11092019/48015322-ced3f300-e120-11e8-9c8f-623685490aa5.png">
<img width="1435" alt="screenshot 2018-11-05 at 17 32 03" src="https://user-images.githubusercontent.com/11092019/48015323-ced3f300-e120-11e8-99e4-dee2232d6c3b.png">

schalkneethling commented on kuma/static/styles/users.scss:
```diff
@@ -13,6 +13,8 @@ Profile view
 @import 'components/activity';
 @import 'components/banned';
+@import 'components/readable-line-length';
+@import 'components/well';
```
I agree with @jwhitlock, could we change this to something a bit more descriptive/semantic @joshJarr?

joshJarr replied:
Sure! This component isn't a `well` anymore, will probably call it something like `alert-message`. 
For context, called it well originally because of the old bootstrap wells https://getbootstrap.com/docs/3.3/components/#wells
Turns out these have been deprecated - wells are no longer fashionable.

joshJarr commented on kuma/users/jinja2/socialaccount/signup.html:
```diff
@@ -35,9 +37,21 @@ <h1>{{ _('Create your MDN profile to continue') }}</h1>
+                  <p>
+                    <h3>{{_('Why do I need to set up an MDN account to complete my subscription?')}}</h3>
+                    {% trans %}
+                    An MDN accounts allows us to communicate with you directly
```
Good spot, I think the copy is still under review, but this is a step in the right direction!

joshJarr commented:
@jwhitlock @schalkneethling I've addressed your feedback here, thanks for reviewing!
If  `bug 1475871` happens to be related to the work we've done let me know and I'm happy to look into the issue - I don't want to introduce new bugs into the existing MDN site.

Thanks.

schalkneethling commented on kuma/users/jinja2/socialaccount/signup.html:
```diff
@@ -12,11 +12,13 @@
 {% set github_connect_url = provider_login_url('github', process='connect', next=request.session.sociallogin_next_url|default('/')) %}
 
+
+
```
Nit: Is there a specific reason for these two empty lines? @joshJarr
